projects/loadable/src/lib/loadable.component.ts

```diff
@@ -93,11 +93,11 @@ export class LoadableComponent implements OnChanges {
     this.loadFn();
   }
 
-  _renderTimeoutTemplate() {
+  _renderTimeoutTemplateOrComponent() {
```

Comment:
just rename this to `_renderTimeout`

Reply:
done. Any of the things mentioned above to do in this PR?